appserver/tests/paas/ext-shared-service-enable-disable-test/src/test/java/org/glassfish/tests/paas/enabledisable/ExtSharedServiceEnableDisableTest.java

```diff
@@ -102,11 +106,12 @@ public void test() throws Exception {
                 String[] startdbArgs = {serverContext.getInstallRoot().getAbsolutePath() +
                         File.separator + "bin" + File.separator + "asadmin" + (OS.isWindows() ? ".bat" : ""), "start-database",
                         "--dbhome" , serverContext.getInstallRoot().getAbsolutePath() + File.separator + "databases","--dbhost",ip_address};
-                ProcessExecutor startDatabase = new ProcessExecutor(startdbArgs);
+                ProcessManager startDatabase = new ProcessManager(startdbArgs);
+                startDatabase.setTimeoutMsec(DATABASE_TIMEOUT);
 
                 try {
                     startDatabase.execute();
-                } catch (ExecException e) {
+                } catch (ProcessManagerException e) {
                     e.printStackTrace();
```

Comment:
Wouldn't be better to fail the test? Or we will leave it on another iteration?

Reply:
These tests are unused now ;-) I fixes it for consistency. So yes, we leave it refactoring ;-)